js/readme.md

```diff
@@ -0,0 +1,4 @@
+# This is a project to keep all my codewars solutions.
```

Comment:
While this readme is fine, it does not look like other readmes :slightly_smiling_face: 

Readmes generally consist of:
1. A title
2. Some description

In your case, there is no concrete title. The description is made up of headers. Let's make it regular text :slightly_smiling_face:

Take a look at readmes of other GitHub projects and try to adjust this one

Reply:
I've made changes